test/libsolidity/smtCheckerTests/external_calls/external_reentrancy_crypto.sol

```diff
@@ -0,0 +1,26 @@
+pragma experimental SMTChecker;
+
+abstract contract D {
+	function d() virtual public;
+}
+
+contract C {
+	bytes data;
+
+	bytes32 kec;
+
+	constructor(bytes memory _data) {
+		data = _data;
+
+		kec = keccak256(data);
+	}
+
+	function f() public view {
```

Comment:
Can you add a new test (or change this) so that `f` takes a parameter `bytes memory _data` and add a failing `assert(_kec == keccak256(_data))`?

Reply:
Failing assert added to this test.